### PR TITLE
Media Recording: Improve settings dialog

### LINKED ISSUE #12312: Media Recording: Improve settings dialog
## Feature Description

[Originally reported here](https://github.com/GoogleForCreators/web-stories-wp/pull/12067#issuecomment-1243751746):

The dialog seems a little confusing.

Let's add visual labels to the dropdowns to make their purpose more clear.

image

Adding labels definitely helps improve accessibility here

### PR DESCRIPTION
**Pull request by timarney**
## Context

The video settings dialog is a  little confusing.

Let's add visual labels to the dropdowns to make their purpose more clear.

## Summary

Add visual labels to the video settings dropdowns

|   Before	|  After  	| 
|---	|---	|
|   before	|  after 	|   

## Testing Instructions

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Press the Record Video / Audio button under the insert panel
2. Press the options button in the editor screen to open the options dialog
3. Video the labels that have been added to the dropdowns

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #12312

### CODE REVIEW

File: packages/story-editor/src/components/mediaRecording/settingsModal.js [hunk 1]
```diff
@@ -174,6 +179,9 @@ function SettingsModal() {
 
       {videoInputs.length > 0 && (
         <Row>
+          <Label size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
+            {__('Video Input', 'web-stories')}
```
Comment (swissspidy) [inline packages/story-editor/src/components/mediaRecording/settingsModal.js]:
```suggestion
            {__('Camera', 'web-stories')}
```

File: packages/story-editor/src/components/mediaRecording/settingsModal.js [hunk 2]
```diff
@@ -186,6 +194,9 @@ function SettingsModal() {
       )}
       {audioInputs.length > 0 && (
         <Row>
+          <Label size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
+            {__('Audio Input', 'web-stories')}
```
Comment (swissspidy) [inline packages/story-editor/src/components/mediaRecording/settingsModal.js]:
```suggestion
            {__('Microphone', 'web-stories')}
```

**Comment by swissspidy** [comment]
Can you add before/after screenshots to the PR description under "user-facing changes" please? Makes it easier to see the difference.

**Review by merapi — APPROVED** [review]

**Review by swissspidy — DISMISSED** [review]
This should be a semantic `<label>` element that is connected to the input via a `for`/`htmlFor` attribute.

With our `Text` component from the design system you can achieve this by doing `<Label as="label">`.

**Comment by spacedmonkey** [comment]
+1 for labels. Labels should be click and that should focus on the dropdown.

**Comment by timarney** [comment]
Updated to use Label `as`
Screen Shot 2022-10-27 at 7 29 44 PM

**Review by spacedmonkey — APPROVED** [review]

**Review by swissspidy — APPROVED** [review]

**Pull request merged**